src/components/MultiProgress/MultiProgress.d.ts

```diff
@@ -0,0 +1,25 @@
+export interface ProgressBarProps {
+
+    /**
+	 * Value of the progress in percentage of linear progress bar component
+	 */
+    value: number;
+
+    /**
+	 * Color of the spinner based on available options
+	 */
+    color: string;
+}
+
+export interface MultiProgressBarProps {
+
+    /**
+     * Configuration array for the multiple progress bars
+     */
+    progressBarConfig: ProgressBarProps[];
+
+    /**
+	 * To override or extend the styles applied to the component
+	 */
+    className: string;
```

Comment:
Fix the formatting issue & don't use `interface`.

Reply:
Check when should we use interface.

Reply:
Fixed